native/nativeCustomProtocolHandler.ts

```diff
@@ -0,0 +1,20 @@
+import lodash from "lodash";
+
+const THROTTLE_TIMEOUT = 1000; // 1 second
+
+export const handleCustomProtocolInvocation = lodash.throttle(
```

Comment:
Not sure why eslint doesn’t complain, but this should have some type annotations.

Reply:
d4b0d75aad8ebefb53837237b34ecbd1c434b098